api-server/README.md

```diff
@@ -78,6 +69,121 @@ properly configured, with the appropriate .env file and pem file in its home dir
 
 run: `./deploy.sh <environment>`
 
+## Development Environment Setup using Docker & docker-compose
+
+Dependencies:
+- [docker](https://www.docker.com)
+- [casey/just](https://github.com/casey/just/releases/)
+
+### Quick Start
+
+These commands will build local docker images and start the environment.  The
+API can be accessed at http://localhost:3003/api
+
+```console
+$ cp .env.sample .env
+$ cp keys/1nfinite.pem 1nfinite.pem
+$ just build start
+```
+
+Notes:
+* A `.env` file is used to set values for many variables. Multiple tools in the
+  toolchain- `just`, `docker-compose`, and `npm` and `node` (via
+  [dotenv](https://www.npmjs.com/package/dotenv)) all use this file. Different
+  values are used for different enviroments.  The `.env.sample` includes variables
+  appropriate for local development.
```

Comment:
Actually for local development the idea is you shouldn't need anything here. That said defaults probably assuming local development on the host. For running inside a docker-compose stack some values need to be overridden but I would do that via container host rather than creating a .env in the container. Mostly the .env.sample exists to document the values you can set